alacritty_terminal/src/term/mod.rs

```diff
@@ -624,22 +629,18 @@ pub struct SizeInfo {
     /// Horizontal window padding.
     pub padding_y: f32,
 
+    /// Number of lines in the viewport.
+    pub screen_lines: Line,
+
+    /// Number of columns in the viewport.
+    pub cols: Column,
+
```

Comment:
I wonder what's the benefit of storing those, instead of computing on demand when needed?

Reply:
It's just more efficient?